xds/src/main/java/com/linecorp/armeria/xds/RouteEntry.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2025 LINE Corporation
```

Comment:
```suggestion
 * Copyright 2025 LY Corporation
```
Time to update our template 😄